### PR TITLE
Ramon 💃 👏 👏

### PR DESCRIPTION
**Pull request by StevendeVries**
Just because...

### CODE REVIEW

**Comment by jeroenvdgulik** [comment]
👍

File: bot.php [hunk 1]
```diff
@@ -27,4 +28,5 @@
 $bot->loadCommand(new Weather());
 $bot->loadCommand(new Cat());
 $bot->loadCommand(new Dog(new FileGetContentsDogUrlFetcher()));
+$bot->loadCommand(new Ramon());
```
Comment (ramondelafuente) [inline bot.php:31]: 👏 👏

File: src/WecampBot/Help.php [hunk 2]
```diff
@@ -22,6 +22,7 @@ protected function execute($message, $context)
 - !weather (Show the weather forecast)
 - !cat (Show a cat. Like duh. Seriously, what did you expect. Why are you still reading?)
 - !dog (Show a dog. Far inferior to !cat. Do not use. Like ever)
+- !ramon (If you really need him)
```
Comment (ramondelafuente) [inline src/WecampBot/Help.php:25]: 👏 👏

File: bot.php [hunk 3]
```diff
@@ -7,6 +7,7 @@
 use W3C\Cat;
 use W3C\Dog;
 use W3C\Help;
+use W3C\Ramon;
```
Comment (ramondelafuente) [inline bot.php:10]: 👏 👏

File: src/WecampBot/Ramon.php [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace W3C;
+
+use PhpSlackBot\Command\BaseCommand;
+
+final class Ramon extends BaseCommand
```
Comment (ramondelafuente) [inline src/WecampBot/Ramon.php:9]: 👏 👏

File: src/WecampBot/Ramon.php [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+{
+    protected function configure()
+    {
+        $this->setName('!ramon');
```
Comment (ramondelafuente) [inline src/WecampBot/Ramon.php:13]: 👏 👏

**Pull request merged**